jans-cedarling/bindings/cedarling_python/cedarling_python.pyi

```diff
@@ -0,0 +1,69 @@
+from typing import *
+
+
+class MemoryLogConfig:
+
+    log_ttl: int
+
+    def __init__(self, log_ttl: int = 60) -> None: ...
+
+
+class StdOutLogConfig:
+    def __init__(self) -> None: ...
+
+
+class DisabledLoggingConfig:
+    def __init__(self) -> None: ...
+
+
+LogConfigs = MemoryLogConfig | StdOutLogConfig | DisabledLoggingConfig
+
+
+class PolicyStoreSource:
+    def __init__(self, json: Optional[str] = None) -> None: ...
+
+
+class PolicyStoreConfig:
+    def __init__(
+        self,
+        source: Optional[PolicyStoreSource] = None,
+        store_id: Optional[str] = None
+    ) -> None: ...
+
+
+class JwtConfig:
+    enabled: bool
+    signature_algorithms: List[str]
+
+    def __init__(
+        self, enabled: bool,
+        signature_algorithms: List[str] = None) -> None: ...
```

Comment:
`None` cannot be assigned to type `List[str]`. If it is an optional value use `Optional[List[str]]` or `List[str] | None`

https://docs.python.org/3/library/typing.html#typing.Union
https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
Furthermore, we want minimum Python 3.10 to be able to use these typings, so lower python versions should be removed from `tox.ini`

Reply:
resolved